### DISCUSSION TITLE
Scope of Digipol

### DISCUSSION THREAD

**Discussion by alexjuiceman** [voteflux/digipol#172] [Q&A] [answered]
What is the intended scope of the digipol app? 

Is the plan to use it as a test bed for the basic digital democracy set up before building something new that incorporates some of the other features of IBDD or would it grow and add those over time?

**Comment by KipCrossing** [comment, marked as answer]
This has been thought about by our leader @okdaithi 

https://github.com/voteflux/flux-tech-roadmap/issues/11

![89120615-4d261a80-d4ea-11ea-9433-17ad64f57313](https://user-images.githubusercontent.com/7298449/102835052-ceeef400-4449-11eb-8346-a1e1c6dcb44c.png)
